presto-product-tests/src/assembly/scripts.xml

```diff
@@ -0,0 +1,28 @@
+<assembly>
+<id>scripts</id>
+<formats>
+    <format>zip</format>
+</formats>
+
+<fileSets>
+   <fileSet>
+        <directory>./</directory>
+        <includes>
+            <include>README.md</include>
+        </includes>
+    </fileSet>
+    <fileSet>
+        <directory>bin</directory>
+    </fileSet>
+    <fileSet>
+        <directory>conf/docker</directory>
+    </fileSet>
+    <fileSet>
+        <directory>conf/presto</directory>
+    </fileSet>
+    <fileSet>
+        <directory>conf/tempto</directory>
+    </fileSet>
```

Comment:
What do you think about including the presto-product-tests jar file here too, so that this zip file contains everything that you need to run the product tests?